### DISCUSSION TITLE
Project Status - Alive and Maintained

### DISCUSSION THREAD

**Discussion by mikaelmello** [mikaelmello/inquire#57] [Announcements]
_New version of the text, for previous ones just check the edit history_

**Last Update: 2023-04-16**

Hi everyone! Almost mid 2023!

Leaving a quick update confirming that **this project still has my full attention**. 

If you have any issues when using it, or features you need, let us know and we can work together to find a solution :)

Some great improvements on the testability and extensibility of inquire are in the works.

**Comment by JK-Kitopi** [comment]
Hi!
Is this project still maintained? Only some minor changes in last months and almost one year since lest release?

**Reply by mikaelmello** [reply]
Hey @JK-Kitopi! Unfortunately it's been really hard to find time for it lately...

**Reply by JK-Kitopi** [reply]
That's bad I really like this project :(

**Reply by mikaelmello** [reply]
I really like it too lol, keep thinking about it every week. Will see if I can separate some time for it over the next month and sweep through the issues/PRs

**Comment by mikaelmello** [comment]
I guess most people should just look at the latest release or commits to figure this out, this has been pinned for a long time when it wasn't really actively maintained :) (It is now!)
